Common/Currencies.cs

```diff
@@ -21,24 +21,46 @@ namespace QuantConnect
     /// <summary>
     /// Provides commonly used currency pairs and symbols
     /// </summary>
-    public static class Currencies
-    {
+    public static class Currencies {
         /// <summary>
         /// Gets the listing of crypto pairs used for currency conversion rates
         /// </summary>
         /// <remarks>
         /// It must include all currency pairs needed to resolve quote currencies in <see cref="Cash.EnsureCurrencyDataFeed"/>
         /// </remarks>
-        public static readonly IReadOnlyList<string> CryptoCurrencyPairs = new List<string>
-        {
+        public static readonly IReadOnlyList<string> CryptoCurrencyPairs = new List<string>() {
```

Comment:
Since there are other fixes to be made, you could undo this change here because of the style we follow.

Reply:
Missed that one, will check everything for code style mismatch again.